src/addons/__tests__/Grid.spec.js

```diff
@@ -231,6 +231,58 @@ describe('Grid', function() {
     });
   });
 
+  describe('Cell Navigation', function() {
+    describe('when cell selection looping is disabled', function() {
+      beforeEach(function() {
+        this.component = this.createComponent({enableCellSelectionLoop: false, enableCellSelect: true});
+      });
+
+      describe('when on last cell', function() {
+        beforeEach(function() {
+          this.component.setState({ selected: { idx: 3, rowIdx: 1 } });
+        });
+        it('selection should move to first cell in next row', function() {
+          this.simulateGridKeyDown('Tab');
+          expect(this.component.state.selected).toEqual({ idx: 0, rowIdx: 2 });
```

Comment:
Didn't understood the main purpose on the first look I'm sorry for that.

Basicly we don't want to completely drop the current functionality, we want to be able to:

1. do what the grid does now,
2. loop in a row
3. move to next/previous row when selection moves beyond last/first cell in row.

The default behaviour should be the current behaviour so that people using the grid at the moment don't end up having unexpected behaviours.

Sorry for the misunderstood.